__tests__/app.test.js

```diff
@@ -382,3 +382,40 @@ describe("/api/comments/:comment_id", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test("POST:201, posts a new article and responds with the article", () => {
+    return request(app)
+      .post("/api/articles")
+      .send({
+        author: "icellusedkars",
+        title: "Welcome to my article",
+        body: "this is an article containing important information about cats",
+        topic: "cats",
+      })
```

Comment:
You would expect this object to have the following properties: 

- username
- body

I'd recommend creating another test to check that it ignores unnecessary properties, for example:
```
{
username: "lurker", 
body: "riveting article",
extraProprties: "extra properties"
}

```